meinberlin/assets/scss/components_user_facing/_live_questions.scss

```diff
@@ -1,46 +1,152 @@
-.live-question {
-    display: flex;
-    flex-direction: column;
-}
-
-.live-question__icon-switch {
-    margin-left: auto;
-    margin-bottom: .625em;
-}
-
-.live-question__progress {
-    display: flex;
-    gap: 8px;
-    margin-bottom: 8px;
-    position: relative;
-}
-
-.live-question__progress-bar {
-    background-color: $gray-lighter;
-    border-radius: 10px;
-    height: 100%;
-    position: absolute;
-    z-index: 0;
-}
-
-.live-question__progress-text {
-    padding: 0 0.5rem;
-    position: relative;
-    z-index: 1;
-}
-
-.live-question__progress-percentage {
-    font-weight: bold;
-}
-
-.live-question__buttons {
-    @media screen and (min-width: $breakpoint-tablet) {
-        display: flex;
-        gap: 1.25rem;
-    }
-}
-
-.live-question__present {
-    min-width: 14rem;
-    width: 100%;
+.live-question {
+    display: flex;
+    flex-direction: column;
+}
+
+.live-question__icon-switch {
+    margin-left: auto;
+    margin-bottom: .625em;
+}
+
+.live-question__progress {
+    display: flex;
+    gap: 8px;
+    margin-bottom: 8px;
+    position: relative;
+}
+
+.live-question__progress-bar {
+    background-color: $gray-lighter;
+    border-radius: 10px;
+    height: 100%;
+    position: absolute;
+    z-index: 0;
+}
+
+.live-question__progress-text {
+    padding: 0 0.5rem;
+    position: relative;
+    z-index: 1;
+}
+
+.live-question__progress-percentage {
+    font-weight: bold;
+}
+
+.live-question__buttons {
+    @media screen and (min-width: $breakpoint-tablet) {
+        display: flex;
+        gap: 1.25rem;
+    }
+}
+
+.live-question__present {
+    min-width: 14rem;
+    width: 100%;
+}
+
+.live-questions__wrapper {

```

Comment:
I can only see _live_questions.scss in in the components folder which is probably not in use anymore but not sure if it can be deleted? but I can´t see _live_question.scss. Am I missing something?

Reply:
yes you are right, I changed live-questions to  live-question now.